pyquil/noise.py

```diff
@@ -683,13 +683,14 @@ def _apply_local_transforms(p: np.ndarray, ts: Iterable[np.ndarray]) -> np.ndarr
     and a list of assignment probability matrices (one for each bit in the readout, ordered like
     the inner axis of results) apply local 2x2 matrices to each bit index.
 
-    :param p: An array that enumerates a function indexed by bitstrings::
+    :param p: An array that enumerates a function indexed by
+        bitstrings::
```

Comment:
Well now that I see this second example, I assume the newline probably is required!

Reply:
I don't fully understand the logic RST uses within these nested structures, but as above, this seemed to make the validator happy - but I didn't have an HTML rendering to verify this with.